src/components/Search/Search.test.tsx

```diff
@@ -87,18 +91,44 @@ describe('Search Component', () => {
 	// 	// see that getSearchOptions (the async) was called 1 time
 	// })
 
-	// it('Should clear the form when clear button clicked', () => {
-	// 	render(<MemoryRouter><Search /></MemoryRouter>)
-	// 	const clearButton = screen.getByRole('button', { name: /clear/i })
-	// 	// need to render some fake options
-	// 	// find one of them (getByText???)
-	// 	// click it
-	// 	// see that state updated with it?
-	// 	// when clear button clicked, see that state is empty?
-	// 	const option1 = screen.getByRole('button', { name: /buttonname/i})
-	// 	fireEvent.click(option1)
-	// 	fireEvent.click(clearButton)
-	// 	expect(clearButton).toBeInTheDocument()
```

Comment:
A fireEvent.change might work here?

Reply:
I had moved this test down to line 113 and it is currently passing but open to other suggestions on how to do it!